src/main/webapp/src/app/app.component.ts

```diff
@@ -13,13 +13,21 @@
 // limitations under the License.
 
 import {Component} from '@angular/core';
+import {Router} from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css', './common/growpod-page-styles.css'],
 })
 export class AppComponent {
-  // For the mockup, the user is always logged in
-  loggedIn = true;
+  constructor(private router: Router) {}
+
+  isLoggedIn(): boolean {
+    if (this.router.url === '/page/login') {
```

Comment:
`return (this.router.url !== 'page/login')`

Reply:
Also let's put `'page/login'` into a named const.

Reply:
done!